app/flatpak-builtins-utils.c

```diff
@@ -819,20 +819,42 @@ get_permission_tables (XdpDbusPermissionStore *store)
 
 /*** column handling ***/
 
+static gboolean
+parse_ellipsize_suffix (const char *p,
+                        FlatpakEllipsizeMode *mode,
+                        GError **error)
+{
+  if (g_str_has_prefix (":full", p))
```

Comment:
This being first makes "app:" mean "app:full", because p=":" is a prefix of ":full". Thats all right though i think.

Reply:
ah, and I thought I could get away with a single comparison :(